GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1115,23 +1115,35 @@ private void RevisionGridSelectionChanged(object sender, EventArgs e)
                 CommitInfoTabControl.SelectedIndexChanged -= CommitInfoTabControl_SelectedIndexChanged;
                 if (!revisions.Any() || GitRevision.IsArtificial(revisions[0].Guid))
                 {
-                    //Artificial commits cannot show tree (ls-tree) and has no commit info 
+                    //Artificial commits cannot show tree (ls-tree) and has no commit info
                     CommitInfoTabControl.RemoveIfExists(CommitInfoTabPage);
                     CommitInfoTabControl.RemoveIfExists(TreeTabPage);
                     CommitInfoTabControl.RemoveIfExists(GpgInfoTabPage);
 
                     if (_showRevisionInfoNextToRevisionGrid)
                     {
                         RevisionsSplitContainer.Panel2Collapsed = true;
-
                     }
+
+                    // Mono special - must re-select the tab for the content to render..
+                    CommitInfoTabControl.SelectedTab = null;
+                    CommitInfoTabControl.SelectedTab = DiffTabPage;
```

Comment:
This will mess up if you have Console (and possibly future tabs) selected before.